hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -1639,7 +1643,14 @@ def complex_numbers(
             f"Cannot have allow_nan={allow_nan!r}, min_magnitude={min_magnitude!r} "
             f"max_magnitude={max_magnitude!r}"
         )
-    allow_kw = {"allow_nan": allow_nan, "allow_infinity": allow_infinity}
+    allow_kw = {
```

Comment:
OK, looks like Mypy thinks that this includes a non-numeric `min_value` 🙄 

Best solution is probably to just drop a `# type: ignore` on line 1659 below.